### PR TITLE
Add memo and fee xlm convertor

### PR DESCRIPTION
**Pull request by Ja-boop**

### CODE REVIEW

**Comment by Ja-boop** [comment]
image
image

File: src/lib/components/transaction/Transaction.svelte [hunk 1]
```diff
@@ -51,6 +51,10 @@
         $operationsVisibility = $operationsVisibility.map(() => $areOperationsExpanded);
     }
 
+    function convertStroopsToXLM(fee: string) {
+        return Number(fee) / 10000000;
```
Comment (Genebson) [inline src/lib/components/transaction/Transaction.svelte]: This should be a constant `10000000`

File: src/lib/i18n/languages/spanish.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "ENGLISH": "Inglés",
     "ERROR": "Error",
     "EXPAND_ALL": "Expandir todas",
-    "FEE": "Comisión:",
+    "FEE": "Comisión del network:",
```
Comment (Genebson) [inline src/lib/i18n/languages/spanish.json]: Comisión de la red.

File: src/lib/i18n/languages/english.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "ENGLISH": "English",
     "ERROR": "Error",
     "EXPAND_ALL": "Expand all",
-    "FEE": "Fee:",
+    "FEE": "Network Fee:",
```
Comment (Genebson) [inline src/lib/i18n/languages/english.json]: "FEE" => "NETWORK_FEE"

**Review by Genebson — APPROVED** [review]

**Pull request merged**